coc/index.html

```diff
@@ -210,12 +210,11 @@
 					<img class="background-image" alt="Background Image" src="../img/hero7.jpg">
 				</div>
 
+				<!-- pagination with <br> creates a blank block to prevent overlapping -->
+				<div class="pagination"><br><br><br><br><br><br><br><br></div>
```

Comment:
I was asking for div class="row", but however, this is also acceptable although is not that much a good standard to keep

Reply:
@abishekvashok 
I'll surely work on that, however currently my mid semester exams are ongoing so as soon as exams get over I'll start working on a more standard solution. 

Thanks

Reply:
This can stay open till your exams are one, no pressure :)